esp/public/media/scripts/onsite/ajax_status.js

```diff
@@ -243,6 +243,10 @@ function update_checkboxes()
             {
 		      studentcheckbox.attr("disabled", "disabled");
             } 
+            else if (data.classes[section.class_id].category__id = open_class_category.id)
```

Comment:
Ah yes, it should. (That's weird, I thought I tested this bit and it worked as intended.)